sips/minutes/_posts/2016-07-15-sip-minutes.md

```diff
@@ -131,17 +135,17 @@ be resolved before resuming its evaluation.
 revisit the support of async/await in try/catch blocks. When that's considered,
 this SIP should be reopened and and it should see another round of discussion.
 
-#### Proposal SIP-20: Improved lazy val initialization by Dmitry Petrashko
+#### Proposal SIP-20: Improved lazy val initialization presented
 Sebastien reviews the SIP and asks Dmitry, present in the meeting, to correct
 him if he's wrong. He agrees that the SIP is desirable but he's unsure about the
 benchmarks and which of the proposals is faster. Dmitry explains that the
-benchmarks are in the repo, and that in the best case the initialization doesn’t
-need synchronization.
+benchmarks are in the repository.
   
-He also recommends the author to include more documentation.
+Sebastien also recommends the authors to include more documentation.
   
-**Outcome**: Dmitry volunteers to provide an implementation in three months. The
-Process Lead schedules the next iteration by October 2016.
+**Outcome**: Dmitry says he's likely to have time to update the SIP in three
```

Comment:
Just because Dmitry is a PhD student at EPFL doesn't mean that he deserves special treatment of having his SIP in some kind active state when it's known it won't really be. It should get the same treatment as the postponed async SIP if there's no plan of actually implementing ths.